src/sinks/elasticsearch/common.rs

```diff
@@ -31,6 +31,7 @@ pub struct ElasticsearchCommon {
     pub base_url: String,
     pub bulk_uri: Uri,
     pub auth: Option<Auth>,
+    pub aws_service: String,
```

Comment:
Maybe this can be an enum instead of string? (to make invalid states unrepresentable)